packages/shared/src/lib/analytics.ts

```diff
@@ -101,7 +101,12 @@ export enum AnalyticsEvent {
   UpvoteSearch = 'upvote search',
   DownvoteSearch = 'downvote search',
   CopySearch = 'copy search',
+  CopyKeyLink = 'copy key link',
+  HideFromHeader = 'hide from header',
+  CloseInvitationPopup = 'close invitation popup',
   ErrorSearch = 'error search',
+  AcceptInvitation = 'accept invitation',
+  SearchHighlightAnimation = 'search highlight animation',
```

Comment:
Taxonomy states `highlight search`

Reply:
Ah, must be recent change. Let me update this too.